coral-spark/src/main/java/com/linkedin/coral/spark/CoralSpark.java

```diff
@@ -72,6 +85,30 @@ public static CoralSpark create(RelNode irRelNode) {
     return new CoralSpark(baseTables, ImmutableList.copyOf(sparkUDFInfos), sparkSQL);
   }
 
+  /**
+   * Users use this function as the main API for getting CoralSpark instance.
+   *
+   * Internally IR RelNode is converted to Spark RelNode, and Spark RelNode to Spark SQL.
+   *
+   * It returns an instance of CoralSpark which contains
+   *  1) Spark SQL
+   *  2) Base tables
+   *  3) Spark UDF information objects, ie. List of {@link SparkUDFInfo}
+   *
+   * @param irRelNode A IR RelNode for which CoralSpark will be constructed.
+   * @param mscClient client interface used to interact with the Hive Metastore service.
+   *
+   * @return [[CoralSparkInfo]]
```

Comment:
nit: it returns CoralSpark not CoralSparkInfo

Reply:
made the change in all javadocs. thanks for pointing it out.